pkg/operator/ceph/object/controller.go

```diff
@@ -52,7 +52,8 @@ import (
 )
 
 const (
-	controllerName = "ceph-object-controller"
+	controllerName    = "ceph-object-controller"
+	RequireObjectCRDs = "REQUIRE_OBJECT_CRDS"
```

Comment:
Suggest using negative grammar instead. It is more clear. Grammatically, if something is "required" it must exist, and if something is "not required" it may or may not exist.

Instead "disable" more clearly states that something should **not** be running.

```suggestion
	DisableObjectCRDs = "DISABLE_OBJECT_CRDS"
```